lib/PuppeteerSharp/Browser.cs

```diff
@@ -58,9 +57,9 @@ public class Browser : IBrowser
                 });
 
             _defaultContext = new BrowserContext(Connection, this, null);
-            _contexts = new ConcurrentDictionary<string, BrowserContext>(contextIds.ToDictionary(
-                contextId => contextId,
-                contextId => new BrowserContext(Connection, this, contextId)));
+            _contexts = new ConcurrentDictionary<string, BrowserContext>(
+                contextIds.Select(contextId =>
+                    new KeyValuePair<string, BrowserContext>(contextId, new(Connection, this, contextId))));
```

Comment:
This avoids constructing a `Dictionary<string, BrowserContext`> only to enumerate it when constructing `ConcurrentDictionary<string, BrowserContext>`